Initial draft of Enterprise Beans 4.1 plan

tadamski opened the pull request:
I'm attaching the draft plan for Enterprise Bean update related to Jakarta EE 12 release.

jhanders34 commented:
@tadamski thank you for creating this draft release plan.  Can you update the description to use the template: https://github.com/jakartaee/specifications/blob/master/plan_review_pr_template.md and update things accordingly for the checklist?  You do not appear to be a committer for Enterprise Beans, so you cannot create a new release on the project page:  https://projects.eclipse.org/projects/ee4j.ejb.  You will need to work with a committer in order to get that updated.

tadamski commented:
@jhanders34 I have changed the title and mentioned lack of backwards incompatible changes. Are there any more changes necessary?

chengfang reviewed (approved):
LGTM

dblevins commented:
Note to the specification committee, the contents of this PR are still under discussion in the EJB spec group.  No plan has yet to be agreed.

tadamski commented:
@jhanders34 I have updated the description and added release record link (but for now it is invalid because I don't have the rights to create release plan).

AngeloRubens commented:
<div style="direction:ltr; font-family:Calibri,Helvetica,sans-serif; font-size:12pt; color:rgb(0,0,0)" data-olk-copy-source="MessageBody">Hi All,</div><div style="direction:ltr; font-family:Calibri,Helvetica,sans-serif; font-size:12pt; color:rgb(0,0,0)"><br></div><pre><div style="direction:ltr; text-align:left; font-family:Calibri,Helvetica,sans-serif; font-size:12pt; color:rgb(0,0,0)">why not think about standardizing this implementation present in wildfly?  like this:</div></pre><div style="direction:ltr; font-family:Calibri,Helvetica,sans-serif; font-size:12pt; color:rgb(0,0,0)"><br></div><ul style="direction:ltr; margin-top:0px; margin-bottom:0px; list-style-type:disc"><li style="font-family:Calibri,Helvetica,sans-serif; font-size:12pt; color:rgb(0,0,0)"><pre><div style="direction:ltr"> Ejb Over http/Http2 </div></pre></li><ul style="direction:ltr; margin-top:0px; margin-bottom:0px; list-style-type:circle"><li style="font-family:Calibri,Helvetica,sans-serif; font-size:12pt; color:rgb(0,0,0)"><div style="direction:ltr"><a id="x_m_860318879387092217m_1410207241206516994m_612839583600332875LPlnk" data-auth="NotApplicable" rel="noopener noreferrer" target="_blank" href="https://github.com/wildfly/wildfly-http-client/blob/c8f7f52a084714b6c5a5f993ebf2f89ac1f52751/docs/src/main/asciidoc/wire-spec-v3.asciidoc" title="https://github.com/wildfly/wildfly-http-client/blob/c8f7f52a084714b6c5a5f993ebf2f89ac1f52751/docs/src/main/asciidoc/wire-spec-v3.asciidoc" data-linkindex="6">https://github.com/wildfly/wildfly-http-client/blob/c8f7f52a084714b6c5a5f993ebf2f89ac1f52751/docs/src/main/asciidoc/wire-spec-v3.asciidoc</a></div><div><div style="width:100%; margin-top:16px; margin-bottom:16px; max-width:800px; min-width:424px" id="x_m_860318879387092217m_1410207241206516994m_612839583600332875LPBorder_GTaHR0cHM6Ly9naXRodWIuY29tL3dpbGRmbHkvd2lsZGZseS1odHRwLWNsaWVudC9ibG9iL2M4ZjdmNTJhMDg0NzE0YjZjNWE1Zjk5M2ViZjJmODlhYzFmNTI3NTEvZG9jcy9zcmMvbWFpbi9hc2NpaWRvYy93aXJlLXNwZWMtdjMuYXNjaWlkb2M.">
  | wildfly-http-client/docs/src/main/asciidoc/wire-spec-v3.asciidoc  at c8f7f52a084714b6c5a5f993ebf2f89ac1f52751 ·  wildfly/wildfly-http-clientThe client libraries that support EJB, Naming and Transactions over HTTP - wildfly/wildfly-http-clientgithub.com
-- | --


</div><div style="direction:ltr"><br></div></li></ul></ul><div style="direction:ltr; font-family:Calibri,Helvetica,sans-serif; font-size:12pt; color:rgb(0,0,0)">Regards</div><div style="direction:ltr; font-family:Calibri,Helvetica,sans-serif; font-size:12pt; color:rgb(0,0,0)">Angelo Rubini</div>Hi All,

why not think about standardizing this implementation present in wildfly?  like this:


     Ejb Over http/Http2 

        https://github.com/wildfly/wildfly-http-client/blob/c8f7f52a084714b6c5a5f993ebf2f89ac1f52751/docs/src/main/asciidoc/wire-spec-v3.asciidoc
        	
        [wildfly-http-client/docs/src/main/asciidoc/wire-spec-v3.asciidoc at c8f7f52a084714b6c5a5f993ebf2f89ac1f52751 · wildfly/wildfly-http-client](https://github.com/wildfly/wildfly-http-client/blob/c8f7f52a084714b6c5a5f993ebf2f89ac1f52751/docs/src/main/asciidoc/wire-spec-v3.asciidoc)
        The client libraries that support EJB, Naming and Transactions over HTTP - wildfly/wildfly-http-client
        [github.com](http://github.com/)

    Distributed EJB Transaction over http/http2

        https://developers.redhat.com/articles/2024/08/29/distributed-transactions-jakarta-ee-10-and-wildfly#
        	
        [Distributed transactions with Jakarta EE 10 and WildFly](https://developers.redhat.com/articles/2024/08/29/distributed-transactions-jakarta-ee-10-and-wildfly#)
        Discover how to implement a distributed transaction solution using Jakara EE Jakarta Enterprise Beans (EJB) in this in-depth guide.
        [developers.redhat.com](http://developers.redhat.com/)

Regards
Angelo Rubini

dblevins commented:
@jhanders34 We've discussed and removed the EJB over HTTP feature.  This is now ready to go up for vote.

tadamski commented:
@jhanders34 yes I'm ok with release plan to move to review and ballot
